test/board_test.exs

```diff
@@ -2,16 +2,22 @@ defmodule BoardTests do
   use ExUnit.Case
 
   test "can create a board" do
-    board = Board.create(3)
+    assert Board.create == [["1", "2", "3"], ["4", "5", "6"],[ "7", "8", "9"]]
+  end
+
+  test "can place a marker on the bord" do
```

Comment:
I noticed that you're not quite consistent with your test names. Sometimes you're saying `it can do something` and sometimes you say `can do something` and leave the `it` out.